### PR TITLE
Adds mixerapi/bake to skeletons section

### PR DESCRIPTION
**Pull request by cnizzardini**
https://github.com/mixerapi/bake

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -377,6 +377,7 @@ Additional lists you might find useful:
 
 - :strawberry: [App template](https://github.com/cakephp/app) - An empty CakePHP project for use with composer.
 - [App template with material ui](https://github.com/coolops/web-app) - CakePHP material ui admin template project for use with composer.
+- :strawberry: [MixerAPI/Bake](https://github.com/mixerapi/bake) - Bake theme for generating RESTful controllers.
```
Comment (dereuromark) [inline README.md:380]: The declared name clearly is not what as per our guidelines the name actually is https://github.com/mixerapi/bake/blob/master/composer.json#L20

Comment (cnizzardini) [reply README.md:380]: Weird, should be `library`, thanks. What exactly do you want me to change the name to?

Comment (dereuromark) [reply README.md:380]: no, it should be "cakephp-plugin", if it is a plugin :)

Comment (cnizzardini) [reply README.md:380]: What about the name of the package? You had concerns about that

Comment (dereuromark) [reply README.md:380]: https://github.com/FriendsOfCake/awesome-cakephp/blob/master/CONTRIBUTING.md#tips-for-creating-composer-packages

**Comment by cnizzardini** [comment]
Fixed name and alphabetical order, also composer type.

**Pull request merged**